packages/mcp-client/package.json

```
@@ -55,6 +55,7 @@
 	"dependencies": {
 		"@huggingface/inference": "workspace:^",
 		"@huggingface/tasks": "workspace:^",
-		"@modelcontextprotocol/sdk": "^1.9.0"
+		"@modelcontextprotocol/sdk": ">=1.11.2",
```

```suggestion
		"@modelcontextprotocol/sdk": "^1.11.2",
```
better to "pin" an exact dependency imo